Monika After Story/game/script-holidays.rpy

```diff
@@ -2378,8 +2378,9 @@ label mas_d25_gift_starter:
 
     m 1suo "Let's see what we have here.{w=0.5}.{w=0.5}.{nw}"
 
-    #Pop the last index so we remove gifts from under the tree as we go
-    $ persistent._mas_d25_gifts_given.pop()
+    #Safe-pop the last index so we remove gifts from under the tree as we go
+    if persistent._mas_d25_gifts_given:
```

Comment:
Add logging here because this is a mismatch that could cause other issues down the line and simply hiding it might make future investigations difficult.

Reply:
Also move this to #8507